src/utils/trust.involved.ts

```diff
@@ -128,6 +164,17 @@ export const postTrustInvolvedPage = (
 
     const typeOfTrustee = req.body.typeOfTrustee;
 
+    if (isReview) {
+      switch (typeOfTrustee) {
+          case TrusteeType.HISTORICAL:
+            return safeRedirect(res, config.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_FORMER_BO_URL);
+          case TrusteeType.INDIVIDUAL:
+            return safeRedirect(res, config.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_INDIVIDUAL_URL);
+          case TrusteeType.LEGAL_ENTITY:
+            return safeRedirect(res, config.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_URL);
```

Comment:
no default case here but a TODO on line 194. I'd say throw a service error as a default for both.